### PR TITLE
allow custom section header in agendalist

### PR DESCRIPTION
**Pull request by zlfnhmd**
Allows creating custom header at the top of each section in AgendaList

### CODE REVIEW

File: src/expandableCalendar/agendaList.js [hunk 1]
```diff
@@ -24,17 +23,19 @@ class AgendaList extends Component {
...
-    /** whether to use moment.js for date string formatting 
+    /** whether to use moment.js for date string formatting
      * (remember to pass 'dayFormat' with appropriate format, like 'dddd, MMM D') */
     useMoment: PropTypes.bool,
     /** style passed to the section view */
-    sectionStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array])
-  }
+    sectionStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array]),
+    /** Allow rendering of a custom header at the top of each section  */
+    customSectionHeader: PropTypes.any,
```
Comment (Inbal-Tish) [inline src/expandableCalendar/agendaList.js:32]: there is no need for an extra prop, AgendaList gets all SectionList props, among them 'renderSectionHeader'. Here you can pass your custom render function.

File: src/expandableCalendar/agendaList.js [hunk 2]
```diff
@@ -170,11 +180,13 @@ class AgendaList extends Component {
         showsVerticalScrollIndicator={false}
         onViewableItemsChanged={this.onViewableItemsChanged}
         viewabilityConfig={this.viewabilityConfig}
-        renderSectionHeader={this.renderSectionHeader}
+        renderSectionHeader={this.CustomSectionHeader ? this.CustomSectionHeader : this.renderSectionHeader}
```
Comment (Inbal-Tish) [inline src/expandableCalendar/agendaList.js:183]: Please move this check to the renderSectionHeader() method where you can safely invoke the custom render method passed in the 'renderSectionHeader' prop (see my first comment)

**Comment by Inbal-Tish** [comment]
@zlfnhmd Hi. Thank you for the PR. Please see my comments and remove your formatting so I can review only the relevant code.

File: src/expandableCalendar/agendaList.js (hunk 1 above)
Comment (zlfnhmd) [reply src/expandableCalendar/agendaList.js]: Good review :+1:

**Comment by zlfnhmd** [comment]
@Inbal-Tish Thanks for the review. I have fixed the review asks, please check it out

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**